raster-foundry/raster-foundry
# Start setting up flyway migrations

## Overview
Add a migration docker container and a migrate script similar to the one we're using in other apps.

### Checklist

- ~Description of PR is in an appropriate section of the [changelog](https://github.com/raster-foundry/raster-foundry/blob/develop/CHANGELOG.md) and grouped with similar changes if possible~ _no functional changes yet_
- ~Styleguide updated, if necessary~
- ~Swagger specification updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Notes
This PR does not contain any real migrations, and does not address how we will migrate from our current migration system to flyway. 

`beforeClean.sql` is necessary for `./scripts/migrate clean` to succeed

## Testing Instructions
- run `./scripts/migrate clean` and verify that the database now has nothing in it using `./scripts/psql`
- run `./scripts/migrate migrate` and verify that there now exists a test table
- run `./scripts/load-development-data` so you can get your database back into a useful state for other things.

Closes https://github.com/raster-foundry/raster-foundry/issues/4910
